pkg/libmachine/persist/filestore.go

```diff
@@ -83,7 +83,6 @@ func (s Filestore) SetExists(name string) error {
 func (s Filestore) Exists(name string) (bool, error) {
 	filename := filepath.Join(s.MachinesDir, name, fmt.Sprintf(".%s-exist", name))
 	_, err := os.Stat(filename)
-	log.Debugf("Checking file: %s", filename)
```

Comment:
This is useful for debugging so I would suggest instead of complete remove better to move it to `os.IsNotExist` block and change message around `file not found: filename`.